src/style.css

```diff
@@ -0,0 +1,24 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star--active {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
please do not use modificator here
due to checklist you need to use stars--N to dedicate active stars
try to use `:nth-child`
you can red about it here: https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

Reply:
Thank you! All done